centaur/src/main/scala/centaur/DockerComposeCromwellConfiguration.scala

```diff
@@ -22,7 +22,7 @@ case class DockerComposeCromwellConfiguration(dockerTag: String, dockerComposeFi
         Array("docker-compose", "-f", dockerComposeFile) ++ command
       }
 
-      private val startCommand = composeCommand("up")
+      private val startCommand = composeCommand("up", "--abort-on-container-exit")
```

Comment:
Other.

This will fast-fail the tests. When any one container fails the docker-compose process will exit. That will keep centaur from pinging an unhealthy cromwell cluster for hours.

It does NOT fix the underlying unhealthiness issue, but will hopefully bring it to folks attention for faster debugging.